src/Stratis.Bitcoin/Consensus/Validators/BlockValidator.cs

```diff
@@ -23,7 +21,20 @@ public interface IHeaderValidator
         /// </summary>
         /// <param name="chainedHeader">The chained header to be validated.</param>
         void ValidateHeader(ChainedHeader chainedHeader);
+    }
 
+    public interface IPartialValidation
+    {
+        /// <summary>
+        /// Partial validation of a block, this will not change any state in the consensus store when validating a block.
```

Comment:
`Partial validation of a block, this will not change any state in the consensus store when validating a block.`
=>
`Schedules a block for background partial validation. Partial validation is the validation that doesn't involve coinview rewinding or updating.`